includes/class-simple-local-avatars.php

```diff
@@ -102,6 +102,7 @@ public function add_hooks() {
 		add_filter( 'pre_option_simple_local_avatars', array( $this, 'pre_option_simple_local_avatars' ), 10, 1 );
 
 		add_action( 'admin_init', array( $this, 'admin_init' ) );
+		add_action( 'init', array( $this, 'define_avatar_ratings' ) );
```

Comment:
@peterwilsoncc is there any reason we are initializing `private $avatar_ratings;` property on action hooks?

Reply:
That was added in 41878022e7a02979ac76f253092b28d7a665c412 by @faisal-alvi for #175. My reading of the commits/pr is that it is to ensure the language files have loaded prior to using the strings.